editor/editor_node.cpp

```diff
@@ -1064,6 +1064,9 @@ void EditorNode::_notification(int p_what) {
 				theme->set_constant("dragging_unfold_wait_msec", "Tree", (float)EDITOR_GET("interface/editor/dragging_hover_wait_seconds") * 1000);
 				theme->set_constant("hover_switch_wait_msec", "TabBar", (float)EDITOR_GET("interface/editor/dragging_hover_wait_seconds") * 1000);
 				editor_dock_manager->update_tab_styles();
+
+				bottom_dock_tab_individual_height = EDITOR_GET("interface/editor/bottom_dock_tab_individual_height");
```

Comment:
This needs a better name it's not very descriptive IMO, something like `interface/editor/bottom_dock_tabs_allow_individual_heights` or negate it to say forcing uniform heights IMO, but regardless it should be `bottom_dock_tabs_individual_heights` if it is kept as is